styles.css

```diff
@@ -0,0 +1,76 @@
+body {
+  background-color: pink;
+  text-align: left;
```

Comment:
This reiterates the [default browser styles](https://browserdefaultstyles.com/) for the `body` element and, therefore, can be removed:
```suggestion
```